### ISSUE TITLE
Remove FTW-Serialize-Element.vi (No Longer Works, Fetch XNode Replaces It)

### ISSUE THREAD

**Issue by jassems** [featherweight/featherweight#113]
FTW-Serialize-Element.vi no longer works, rather it throws a run-time error as show below.  This is likely because the new JSON Fetch xnode will serialize any element by simply specifying the "Element Type and Default Value" input as a string.

The Problem:
<img width="1219" alt="screen shot 2017-06-21 at 1 35 48 pm" src="https://user-images.githubusercontent.com/7451453/27402076-07586f84-568b-11e7-9a65-095410cdc491.png">

The Solution:
<img width="1374" alt="screen shot 2017-06-21 at 2 15 48 pm" src="https://user-images.githubusercontent.com/7451453/27402412-2af62f48-568c-11e7-9d4c-efaf7e3db09b.png">

**Comment by JackDunaway** [comment]
Good catch, and thank you for the correct workaround! `Serialize Element` indeed is deprecated, and needs to be removed altogether.

It's worth also pointing out the power of your example: the `Serialize Element` feature may now be accomplished via arbitrary JSON path, rather than just JSON key name within the top level object only provided by the old `Serialized Element` function.

Leaving this issue open until it's properly removed from public API. In meantime, _caveat coder_.
